src/app/templates/app/submit_new_license.html

```diff
@@ -459,11 +459,23 @@
                   }
                 }
                 else{
-                  var warningMessage = "Please note that there was a problem opening the issue for the SPDX legal team. Please email spdx-legal@lists.spdx.org with SPDX ID for the license you are submitting";
-                  $("#messages").html('<div class="alert alert-warning alert-dismissable fade in"><a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a><strong>Warning! </strong>'+ warningMessage +'</div>');
-                      setTimeout(function() {
-                        $("#messages").html("");
-                      }, 7000);
+                  var errorMessage = 'Error occured while submitting the license. Please contact to spdx-legal@lists.spdx.org with SPDX ID for submitting the license. Would you like to submit this problem to github so that the developers can look at it?';
```

Comment:
`Please contact spdx-legal@lists.spdx.org`

Reply:
Can we have constant variable with these message which can be referenced here?

Reply:
Could you elaborate a bit please?

Reply:
Having all these string message in one file and then referencing here in javascript.